boskos/janitor/janitor.py

```diff
@@ -168,6 +175,11 @@ def clear_resources(project, cols, resource, rate_limit):
             else:
                 base.append('--global')
 
+        # hard code asia-southeast1-a for NEG
+        # TODO: remove this once limitation is dropped
```

Comment:
nit: `TODO(freehan):`

and please open an issue to track this

Reply:
okay